app/graph/types/project_media_type.rb

```diff
@@ -206,7 +206,7 @@ def flags
   field :tags, TagType.connection_type, null: true
 
   def tags
-    object.get_annotations('tag').map(&:load)
+    object.get_annotations('tag').map(&:load).sort_by { |tag| tag.tag_text.downcase }
```

Comment:
This is OK for this case because this method already returns an array (see that `.map()` is used)... so `sort_by`, which sorts an array in memory, won't create a performance issue or break pagination.